src/lib-voyager.tsx

```diff
@@ -77,7 +77,8 @@ export class Voyager {
    * @memberof Voyager
    */
   public setSpec(spec: Object) {
-    this.data = undefined;
+    const data = "data";
+    this.data = spec[data];
```

Comment:
I think we should track where they do JSON validation and consider if we should move the logic to here.